### DISCUSSION TITLE
[Solved] Accidentally pushed changed to a wrong repo

### DISCUSSION THREAD

**Discussion by saasforge** [community/community#23307] [New to GitHub]
<p>My local repo is connected to 2 repos: one “official” to pull only from, and the second one - “original” to push changes.</p>
<p>I accidentally pushed changes from my local repo to the original. These changes should not be there.</p>
<p>I tried:</p>
<p>git revert HEAD -m 1</p>
<p>git push origin master</p>
<p>And I see now master [xx0e1] yy50c3eRevert </p>
<p>but the repo I accidentally pushed too looks the same with all the latest changes.</p>
<p>If there is any way to change completely remove the last 2-3 commits?</p>
<p>Oh, I finally solved this problem:</p>
<p>git reset --hard xxxx</p>
<p>git push origin master --force</p>

**Comment by lesleyanneb** [comment, marked as answer]
<p>Hi there,</p>
<p>It sounds like you managed to solve your trouble and using <code>git reset &lt;last good SHA&gt;</code> or <code>git reset --hard &lt;last good SHA&gt;</code> is the best way to go about it. </p>
<p>In case you’re interested, we also have a blog post that explains how to undo (almost) anything with Git via the command line:</p>
<p><a href="https://github.com/blog/2019-how-to-undo-almost-anything-with-git" rel="nofollow noopener">https://github.com/blog/2019-how-to-undo-almost-anything-with-git</a></p>
<p>I hope that helps!</p>
